docs/books/computer-games/computer-battle-games/escape/escape.py

```diff
@@ -0,0 +1,56 @@
+import os, random
+
+os.system('cls||clear')
```

Comment:
There's another way to tackle this, using ANSI sequences for the TTY - see [Beat the Bug Eyes](https://github.com/pybee/yorkshire4/blob/master/docs/books/computer-games/computer-space-games/beat_the_bug_eyes/beat_the_bug_eyes.py#L46) for an example. 

Just copy-pasting the clear method is fine for now; but ideally, utilities like `clear()`, `getch()`, etc would be factored out into a support library, and we'd have some docs on how to create a virtual environment and install the support library.